ofl/notoseriftoto/METADATA.pb

```diff
@@ -0,0 +1,27 @@
+name: "Noto Serif Toto"
+designer: "Google"
+license: "OFL"
+category: "SERIF"
+date_added: "2022-09-05"
+fonts {
+  name: "Noto Serif Toto"
+  style: "normal"
+  weight: 400
+  filename: "NotoSerifToto[wght].ttf"
+  post_script_name: "NotoSerifToto-Regular"
+  full_name: "Noto Serif Toto Regular"
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/toto)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/notofonts/toto.git"
+  archive_url: "https://github.com/notofonts/toto/releases/download/NotoSerifToto-v2.001/NotoSerifToto-v2.001.zip"
+}
+is_noto: true
```

Comment:
@simoncozens Missing subsets and languages?

Reply:
We don't have a language definition or sample text for it. :-/

Reply:
Ah, then we can't really merge it, without a specific subset it will only serve Latin, which is a bit useless :/

Reply:
I'll work on creating one in gflanguages.